SpecFlowProject1/StepDefinitions/LanguageActionsStepDefinitions.cs

```diff
@@ -30,8 +30,15 @@ public void WhenUserClicksOnAddNewButtonUnderLanguages()
             actions.AddLanguages(webdriver);
             
         }
+        [When(@"user enters language andproficiency level '([^']*)'")]
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors
        LoginMars login = new LoginMars();
        IWebDriver webdriver =new EdgeDriver();
        AddnEditandDeletelanguages actions = new AddnE